iroh.pyi

```diff
@@ -0,0 +1 @@
+from .iroh_ffi import *  # NOQA
```

Comment:
By adding this, https://www.maturin.rs/project_layout will add the py.typed file to the distribution

Reply:
cool, thanks